### PR TITLE
TE-8944: Get rid of sales_order_item update for fk_sales_shipment

### PR DESCRIPTION
**Pull request by olhalivitchuk**
Ticket: [https://spryker.atlassian.net/browse/TE-8944](https://spryker.atlassian.net/browse/TE-8944)

### CODE REVIEW

File: Sales/11.23.0/installer-manifest.json [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+            "poistion": {
+                "before": "\\Spryker\\Zed\\Shipment\\Communication\\Plugin\\Checkout\\SalesOrderShipmentSavePlugin",
+                "after": "\\Spryker\\Zed\\Sales\\Communication\\Plugin\\Checkout\\OrderSaverPlugin"
+            }
+        },
+        {
+            "target": "\\Spryker\\Zed\\Checkout\\CheckoutDependencyProvider::getCheckoutOrderSavers",
+            "source": "\\Spryker\\Zed\\Sales\\Communication\\Plugin\\Checkout\\OrderItemsSaverPlugin",
+            "poistion": {
+                "before": "\\Spryker\\Zed\\SalesProductConfiguration\\Communication\\Plugin\\Checkout\\ProductConfigurationOrderSaverPlugin",
```
Comment (gechetspr) [inline Sales/11.23.0/installer-manifest.json:23]: ProductConfigurationOrderSaverPlugin might be not present in the plugins list. Not sure how integrator will work in this case

Just to be on the safe side, I would remove this before section

File: Sales/11.23.0/installer-manifest.json [hunk 2]
```diff
@@ -0,0 +1,34 @@
+{
+    "wire-plugin": [
+        {
+            "target": "\\Spryker\\Zed\\Checkout\\CheckoutDependencyProvider::getCheckoutOrderSavers",
+            "source": "\\Spryker\\Zed\\Sales\\Communication\\Plugin\\Checkout\\OrderSaverPlugin",
+            "poistion": {
+                "before": "\\Spryker\\Zed\\Sales\\Communication\\Plugin\\Checkout\\OrderTotalsSaverPlugin",
+                "after": ""
```
Comment (gechetspr) [inline Sales/11.23.0/installer-manifest.json:8]: can be removed

File: Sales/11.23.0/installer-manifest.json (hunk 1 above)
Comment (olhalivitchuk) [reply Sales/11.23.0/installer-manifest.json]: Done

File: Sales/11.23.0/installer-manifest.json (hunk 2 above)
Comment (olhalivitchuk) [reply Sales/11.23.0/installer-manifest.json:8]: Done

**Review by gechetspr — APPROVED** [review]

**Review by demkos — APPROVED** [review]

**Pull request merged**